index.html

```diff
@@ -4,15 +4,204 @@
     <title>Setup and Mobile first</title>
 </head>
 <body>
-    <div class="container">
-        <div class="box">B 1</div>
-        <div class="box one">B 2</div>
-        <div class="box">B 3</div>
-        <div class="box">B 4</div>
+<header>
+    <div class="top-logo">
+        <a href="#">Abbas</a>
     </div>
+    <nav class="Navbar">
+        <div class="menu-icon">
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+        </div>
+    </nav>
+</header>
+        <section class="main">
+            <div class="top-heading">
+                <h1>Hello, My name is Abbas Glad to see you!</h1>
+            </div>
+            <div class="heading-description">
+                <p>I'm a web developer! I can help you to create a Product or Portfolio website for you. Check out my website and contact me if you find it great.</p> 
+            </div>
+            <div class="touch-links">
+                <h3><a href="#">LET'S CONNECT</a></h3>
+                <div class="social-bar">
+                    <ul>
+                        <li><a href="https://twitter.com/Abbas_sDev" class="connect"><img src="image-resources/twitter.png" alt="Twitter logo"></a></li>
+                        <li><a href="https://www.linkedin.com/in/abbas-sarwar-4a0b16257/" class="connect"><img src="image-resources/linkedIn.png" alt="LinkedIn logo"></a></li>
+                        <li><a href="https://github.com/AbbasSarwar" class="connect"><img src="image-resources/Github.png" alt="Github logo"></a></li>
+                        <li><a href="#" class="connect"><img src="image-resources/M.png" alt=" logo"></a></li>
+                        <li><a href="#" class="connect"><img src="image-resources/new.png" alt=" logo"></a></li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+
+        <section class="work-section">
+            <div class="work-imgs">
+                <div class="work-bg"></div>
+            </div>
+            <div class="work-topic">
+                <h2>Tonic</h2>
+            </div>
+            <div class="work-details">
+                <ul>
+                    <li>∘ CANOPY</li>
+                    <li class="work-2nd">∘ BACK END DEV</li>
+                    <li class="work-2nd">∘ 2023</li>
+                </ul>
+            </div>
+            <div class="work-paragraph">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-langs">
+                 <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+                 </ul>
+            </div>
+            <div class="work-button">
+                <button><span>See Project</span></button>
+            </div>
+        </section>
```

Comment:
- [ ] Please, kindly ensure that **all** your design specifications (padding, margin, font-size, font-weight, border-radius and so on) align with that of Figma as much as possible, So as to have a pixel perfect design as you're going to soon be doing in your future job.

- Your design
<img width="156" alt="wrong" src="https://user-images.githubusercontent.com/41163571/219496791-244b907e-6793-40c6-b823-ae840bd7db62.png">

- Figma design
<img width="266" alt="right" src="https://user-images.githubusercontent.com/41163571/219496881-ca4db9ca-5334-4190-9ea8-5f457ab01e25.png">